vault.js

```
@@ -0,0 +1,638 @@
+   * It changes the settings of the vault with the given.
+   * It works in both locked and unlocked states.
+   * @param {Object} settings 
+   */
+  async changeKeys(keys) {
+    if (!this.isSecure()) {
+      return;
+    }
+
+    this._persistSettings(keys);
+    this._sessionStorage.set(this._getKey('settings'), settings);
+  }
```

Biome (2.1.2)

[error] Duplicate class member name "changeKeys"



(lint/suspicious/noDuplicateClassMembers)